pkgs/by-name/kc/kcl/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "kcl";
+  version = "0.8.5";
+
+  src = fetchFromGitHub {
+    owner = "kcl-lang";
+    repo = "cli";
+    rev = "v${version}";
+    hash = "sha256-ZjEMgQukhBGY3LWqsGmLj3lKfLtNNaQugQs0cSLMb80=";
+  };
+
+  vendorHash = "sha256-jmqKMB85HxAlwH7FVjHrLCZQYuAJrguRfzIz1yMypjw=";
+
+  ldflags = [
+    "-X=kcl-lang.io/cli/pkg/version.version=${version}"
+  ];
+
+  subPackages = [ "cmd/kcl" ];
+
+  meta = with lib; {
+    description = "The KCL Command Line Interface (CLI";
+    homepage = "https://github.com/kcl-lang/cli";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ ];
```

Comment:
Consider [adding yourself as a maintainer](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md). Otherwise, lgtm.

Reply:
> Consider [adding yourself as a maintainer](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md). Otherwise, lgtm.

Here's the maintainer PR: https://github.com/NixOS/nixpkgs/pull/302497